README.md

```diff
@@ -125,28 +125,28 @@ spec:
 > Note: Bundles are considered immutable once they are created. See the [bundle immutability doc](/docs/bundle-immutability.md)
 > for more information.
 
-### BundleInstance
+### BundleDeployment
```

Comment:
Hmm -- I see where you're coming from, but I'm not sure we have the userbase right now such that renaming a CRD like this would warrant an explicit mention on the README. It would potentially create some confusion, for a marginal benefit. I think we should make a big callout on the v0.6 milestone about this breaking change, for sure.